working-groups/query-standardization/dsl-survey/ppl/README.md

```diff
@@ -0,0 +1,256 @@
+    * **Weaknesses**: May lack some specific functionalities of other DSLs.
+
+1. **Intended for At-Rest and/or Streaming Data**:
+
+    * Both.
+
+1. **Source Specification**:
+
+    * Users can specify indices or aliases or generic data-sources that abstract the tables as a logical raw data .
+
+1. **Joining with Other Sources**:
+
+    * Joining multiple indices is supported; joining with external sources like CSV is not yet supported.
+
+1. **Differing Retentions in Time Series Sources**:
+
+    * Query federation from Prometheus is supported
+    * This would depend on the OpenSearch data management setup.
+
+1. **Coupling to the Data Store Model (1 to 10)**:
+
+    * 4, It's designed for OpenSearch but with some level of abstraction.
+
+1. **Flexibility Across Different Storage Formats or Contexts**:
+
+    * PPL is specific to OpenSearch and the underlying data structures it supports.
+    * Work in progress to allow data read from object storage
+
+1. **Character Sets Supported**:
+
+    * UTF-8.
+
+1. **Special or Reserved Characters**:
+
+    * Special characters like parentheses, commas, etc., have specific uses in PPL.
+
+1. **Writing Data or Configuring the Backing Store**:
+
+    * PPL is for querying only.
+
+* * *
+
+### **Metrics**
+
+* **Supported Metric Measurement Types**:
+    * Gauges, rates, counters, etc. PPL can work with different metric types both in OpenSearch and in Prometheus .
+* **Handling Changes to Measurement Type Over Time**:
+    * Changes in measurement type are handled at the data level; PPL queries the data as-is without managing type changes.
+* **Supported Values as Metrics**:
+    *  Double precision floating-point, integers, booleans, summary statistics, histogram bucket counts, etc.
+* **Temporal Aggregation**:
+    *  Supports custom aggregation with functions like
+        * [COUNT](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#count)
+        * [SUM](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#sum)
+        * [AVG](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#avg)
+        * [MAX](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#max)
+        * [MIN](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#min)
+        * [VAR_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-samp)
+        * [VAR_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-pop)
+        * [STDDEV_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-samp)
+        * [STDDEV_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-pop)
+        * [TAKE](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#take)
+* **Timestamp Alignment**:
+    * Typically aligned to the start of the window.
```

Comment:
Could you elaborate on `typically` please? We're looking for edge cases in particular.